nsis/tesseract.nsi

```diff
@@ -0,0 +1,1525 @@
+    AddSize 1370
+    !insertmacro Download_Lang_Data ara
+  SectionEnd
+
+  Section /o "Assamese" SecLang_asm
+    AddSize 1950
+    !insertmacro Download_Lang_Data asm
+  SectionEnd
+
+  Section /o "Azerbaijani" SecLang_aze
+    AddSize 3360
+    !insertmacro Download_Lang_Data aze
+  SectionEnd
+
+  Section /o "Azerbaijani (Cyrilic)" SecLang_aze_cyrl
+    AddSize 1850
+    !insertmacro Download_Lang_Data aze_cyrl
+  SectionEnd
+  
+  Section /o "Belarusian" SecLang_bel
+    AddSize 3520
+    !insertmacro Download_Lang_Data bel
+  SectionEnd
+
+  Section /o "Bengali" SecLang_ben
+    AddSize 836
+    !insertmacro Download_Lang_Data ben
+  SectionEnd
+
+  Section /o "Tibetan" SecLang_bod
+    AddSize 1880
+    !insertmacro Download_Lang_Data bod
+  SectionEnd
+
+  Section /o "Bosnian" SecLang_bos
+    AddSize 2380
+    !insertmacro Download_Lang_Data bos
+  SectionEnd
+  
+  ####Welche Sprache ist bre?####
+  
+  Section /o "?????" SecLang_bre
+    #AddSize ????
+    !insertmacro Download_Lang_Data bre
+  SectionEnd
+  
+  Section /o "Bulgarian" SecLang_bul
+    AddSize 1600
+    !insertmacro Download_Lang_Data bul
+  SectionEnd
+
+  Section /o "Catalan" SecLang_cat
+    AddSize 1090
+    !insertmacro Download_Lang_Data cat
+  SectionEnd
+
+  Section /o "Cebuano" SecLang_ceb
+    AddSize 699
+    !insertmacro Download_Lang_Data ceb
+  SectionEnd
+  
+   Section /o "Czech" SecLang_ces
+    AddSize 3620
+    !insertmacro Download_Lang_Data ces
+  SectionEnd
+  
+  Section /o "Chinese (Simplified)" SecLang_chi_sim
+    AddSize 2350
+    !insertmacro Download_Lang_Data chi_sim
+  SectionEnd
+  
+  Section /o "Chinese (Simplified vertical)" SecLang_chi_sim_vert
+    AddSize 1840
+    !insertmacro Download_Lang_Data chi_sim_vert
+  SectionEnd
+  
+  Section /o "Chinese (Traditional)" SecLang_chi_tra
+    AddSize 2260
+    !insertmacro Download_Lang_Data chi_tra
+  SectionEnd
+  
+  Section /o "Chinese (Traditional vertical)" SecLang_chi_tra_vert
+    AddSize 1740
+    !insertmacro Download_Lang_Data chi_tra_vert
+  SectionEnd
+  
+  Section /o "Cherokee" SecLang_chr
+    AddSize 366
+    !insertmacro Download_Lang_Data chr
+  SectionEnd
+  
+  Section /o "Corsican" SecLang_cos
+    AddSize 2190
+    !insertmacro Download_Lang_Data cos
+  SectionEnd
+  
+  Section /o "Welsh" SecLang_cym
+    AddSize 2110
+    !insertmacro Download_Lang_Data cym
+  SectionEnd
+  
+  Section /o "Danish" SecLang_dan
+    AddSize 2460
+    !insertmacro Download_Lang_Data dan
+  SectionEnd
+  
+  Section /o "German" SecLang_deu
+    AddSize 1450
+    !insertmacro Download_Lang_Data deu
+  SectionEnd
+  
+  Section /o "Divehi" SecLang_div
+    AddSize 1690
+    !insertmacro Download_Lang_Data div
+  SectionEnd
+  
+  Section /o "Dzongkha" SecLang_dzo
+    AddSize 439
+    !insertmacro Download_Lang_Data dzo
+  SectionEnd
+
+  Section /o "Greek" SecLang_ell
+    AddSize 1350
+    !insertmacro Download_Lang_Data ell
+  SectionEnd
+  
+  Section /o "English" SecLang_eng
```

Comment:
English is a special case: it is already part of the default installation, so there is no need to add it to the optional installations. If we want to have it there nevertheless, the section label `SecLang_eng` must be made unique.